_posts/molecules/2019-08-08-tooltips.html

```diff
@@ -0,0 +1,13 @@
+---
+layout: post
+title: Tooltips
```

Comment:
Isn't this more like `Help` or `Context sensitive help`?

Reply:
I first thought `Legend` would be nice, then went for `Tooltips`, but it's true that it's way too generic. I like `Context sensitive help`.